### PR TITLE
Current development state

### PR DESCRIPTION
**Pull request by ipa-rmb-pz**

### CODE REVIEW

File: robotino_calibration/ros/src/camera_base_calibration_marker.cpp [hunk 1]
```diff
@@ -216,20 +206,20 @@ bool CameraBaseCalibrationMarker::moveRobot(const calibration_utilities::RobotCo
...
-	msg.data = robot_configuration.pan_angle_;
-	calibration_interface_->assignNewCamaraPanAngle(msg);
-	//pan_controller_.publish(msg);
-	msg.data = robot_configuration.tilt_angle_;
-	//tilt_controller_.publish(msg);
-	calibration_interface_->assignNewCamaraTiltAngle(msg);
+	std_msgs::Float64MultiArray angles;
 	
+	angles.data.resize(2);
+	angles.data[0] = robot_configuration.pan_angle_;
```
Comment (ipa-rmb-pz) [inline robotino_calibration/ros/src/camera_base_calibration_marker.cpp:212]: Why is robot_configuration not changed to contain a variable sized array of pan/tilt/etc. angles?

File: robotino_calibration/ros/src/robotino_interface.cpp [hunk 2]
```diff
@@ -92,44 +104,47 @@ RobotinoInterface::~RobotinoInterface()
 //Callbacks - User defined
 void RobotinoInterface::panJointStateCallback(const dynamixel_msgs::JointState::ConstPtr& msg)
```
Comment (ipa-rmb-pz) [inline robotino_calibration/ros/src/robotino_interface.cpp:105]: Change `dynamixel_msgs::JointState` to `sensor_msgs::JointState`

File: robotino_calibration/ros/src/robotino_interface.cpp [hunk 3]
```diff
@@ -92,44 +104,47 @@ RobotinoInterface::~RobotinoInterface()
...
+	angle.data = newAngles.data[0];
+	pan_controller_.publish(angle);
+	angle.data = newAngles.data[1];
+	tilt_controller_.publish(angle);
 }
 
-double RobotinoInterface::getCurrentCameraPanAngle()
+std::vector<double>* RobotinoInterface::getCurrentCameraState()
 {
-	boost::mutex::scoped_lock lock(pan_tilt_joint_state_data_mutex_);
```
Comment (ipa-rmb-pz) [inline robotino_calibration/ros/src/robotino_interface.cpp:131]: Why was the mutex deleted, should be used.

**Comment by ipa-rmb** [comment]
Already updated on my side - see reverse PR https://github.com/ipa-rmb-mr/squirrel_calibration/pull/1 .

**Pull request closed without merging**